packages/config/.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "node": "7"
```

Comment:
Right now we target Node v6.0.0 and up: https://github.com/expo/expo-cli/blob/7da18891ed56a2c2b323ec0a7ea72d96bf5b7e9f/packages/expo-cli/bin/expo.js#L10-L22

However, 6.x LTS end of life is on April 30th, so we could then drop it and support 8.x and up.